config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   root "welcome#index"
+
+  get '/foods', to: 'foods#show'
```

Comment:
I would think of this route more as an `index` because it returns a list of foods rather than just one food.